Assets/SEE/UI/Window/PropertyWindow/PropertyWindow.cs

```diff
@@ -0,0 +1,845 @@
+            if (!string.IsNullOrEmpty(searchField.text) && !string.IsNullOrWhiteSpace(searchField.text))
+            {
+                ActivateMatches(searchField.text);
+            }
+        }
+        #endregion
+
+        /// <summary>
+        /// Sets the activity of the game objects of the given dictionary (<paramref name="objects"/>)
+        /// </summary>
+        /// <param name="objects">The objects to set their activity.</param>
+        /// <param name="activate">The activity</param>
+        private void SetActive(Dictionary<string, (string, GameObject)> objects, bool activate)
+        {
+            foreach ((_, GameObject go) in objects.Values)
+            {
+                go.SetActive(activate);
+
+                if (activate)
+                {
+                    AnimateIn(go);
+                }
+                else
+                {
+                    if (GameFinder.FindChild(go, "Expand Icon") != null)
+                    {
+                        RotateExpandIcon(go, false);
+                    }
+                }
+
+                if (expandedItems.Contains(go.name))
+                {
+                    SetActive(GetDictOfGroup(go.name), activate);
+                    RotateExpandIcon(go, activate);
+                }
+            }
+            return;
+
+            /// Expands the game object by animating its scale.
+            static void AnimateIn(GameObject go)
+            {
+                go.transform.localScale = new Vector3(1, 0, 1);
+                go.transform.DOScaleY(1, duration: 0.5f);
+            }
+        }
+
+        /// <summary>
+        /// Creates the property window.
+        /// </summary>
+        public void CreateUIInstance()
+        {
+            /// Instantiate PropertyWindow
+            GameObject propertyWindow = PrefabInstantiator.InstantiatePrefab(WindowPrefab, Window.transform.Find("Content"), false);
+            propertyWindow.name = "Property Window";
+
+            items = (RectTransform)propertyWindow.transform.Find("Content/Items");
+            searchField = propertyWindow.transform.Find("Search/SearchField").gameObject.MustGetComponent<TMP_InputField>();
+
+            searchField.onSelect.AddListener(_ => SEEInput.KeyboardShortcutsEnabled = false);
+            searchField.onDeselect.AddListener(_ => SEEInput.KeyboardShortcutsEnabled = true);
+
+            ButtonManagerBasic filterButton = propertyWindow.transform.Find("Search/Filter").gameObject.MustGetComponent<ButtonManagerBasic>();
+            ButtonManagerBasic sortButton = propertyWindow.transform.Find("Search/Sort").gameObject.MustGetComponent<ButtonManagerBasic>();
+            ButtonManagerBasic groupButton = propertyWindow.transform.Find("Search/Group").gameObject.MustGetComponent<ButtonManagerBasic>();
+            PopupMenu.PopupMenu popupMenu = gameObject.AddComponent<PopupMenu.PopupMenu>();
+            UnityEvent rebuild = new();
+            rebuild.AddListener(() => Rebuild());
```

Comment:
I think this can be shortened a bit:
```suggestion
            rebuild.AddListener(Rebuild);
```